src/ctap/main_hid.rs

```diff
@@ -42,7 +42,7 @@ impl<E: Env> MainHid<E> {
             | CtapHid::<E>::CAPABILITY_NMSG;
 
         let hid = CtapHid::<E>::new(capabilities);
-        let wink_permission = None;
+        let wink_permission = <<E as Env>::Clock as Clock>::Timer::default();
```

Comment:
I'm surprised it doesn't work to write `E::Clock::Timer`, or at least `<E::Clock as Clock>::Timer`. That's a bit sad.

Reply:
Actually the latter works for E, but not for i.e. `TestEnv`, so we still need `<<TestEnv as Env>::Clock as Clock>`, but I updated the rest.